pkg/cmd/dev/io/exec/exec.go

```diff
@@ -187,9 +187,12 @@ func (e *Exec) commandContextInheritingStdStreamsImpl(
 
 	_, err := e.Next(command, func(outTrace, errTrace io.Writer) (string, error) {
```

Comment:
`s/outTrace/_` and `s/errTrace/_` given they're unused, and also bump DEV_VERSION?